root/etc/cont-init.d/98-vuetorrent

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/with-contenv bash
+APP_FILEPATH='/config/qBittorrent/qBittorrent.conf'
+RELEASE_FILEPATH='/vuetorrent/release.json'
+NEEDS_UPDATE=false
+CURRENT_RELEASE=$(curl -s https://api.github.com/repos/WDaan/VueTorrent/releases/latest | jq -r ".assets[]")
+
+# ===============================================
+install_deps() {
+    if ! [[ -x "$(command -v jq)" ]]; then
+        echo '-------------------------'
+        echo '| Installing jq package |'
+        echo '-------------------------'
+        apt-get update &&
```

Comment:
Please move this out to a separate init script called `95-apt-get` like here: https://github.com/linuxserver/docker-mods/blob/code-server-awscli/root/etc/cont-init.d/95-apt-get